src/components/ThemeProvider/theme.ts

```diff
@@ -0,0 +1,23 @@
+import type { DefaultTheme } from "styled-components";
+
+const defaultTheme: DefaultTheme = {
+  colorPrimary: "#000000",
+  colorLink: "#1c64ff",
+  colorBgContainer: "#f0f0f0",
+  colorBgElevated: "#e5e5e5",
+  colorText: "#000000",
+  colorTextSecondary: "#888888",
+  colorWhite: "#ffffff",
+};
+
+export const lightTheme: DefaultTheme = {
+  ...defaultTheme,
+};
+
+export const darkTheme: DefaultTheme = {
+  ...defaultTheme,
+  colorText: "#ffffff",
+  colorBgContainer: "#000000",
+  colorBgElevated: "#333333",
```

Comment:
I'd think that all but colorWhite also need adapting here i. e. also `colorTextPrimary` and `colorTextSecondary`.

Reply:
Done